irohad/ordering/impl/ordering_service_transport_grpc.cpp

```diff
@@ -50,18 +58,23 @@ void OrderingServiceTransportGrpc::publishProposal(
         grpc::CreateChannel(peer, grpc::InsecureChannelCredentials()));
   }
 
-  protocol::Proposal pb_proposal;
-  pb_proposal.set_height(proposal.height);
-  for (const auto &tx : proposal.transactions) {
-    new (pb_proposal.add_transactions())
-        protocol::Transaction(factory_.serialize(tx));
-  }
+  auto pb_proposal =
+      shared_model::proto::ProposalBuilder()
+          .height(proposal.height())
+          .createdTime(iroha::time::now())
+          .transactions(proposal.transactions()
+                        | boost::adaptors::transformed([](auto iface_tx) {
+                            std::unique_ptr<iroha::model::Transaction> old_tx(
+                                iface_tx->makeOldModel());
```

Comment:
I think there is forbidden to use `makeOldModel` because u create the new-style proposal.

Reply:
I've described issues in pr message. There's an issue for converting iface->proto